lib/rtoolsHCK.rb

```diff
@@ -1221,8 +1221,6 @@ def reconnect
   def close_for_reconnect
     unload_toolshck
     unload_winrm_ps
-  rescue StandardError => e
```

Comment:
You are also removing the exception handling here, so the commit message is misleading.
But also, why do it?

Reply:
Oh sorry this change should have been in a different commit.
This change removes the unnecessary code, this exception handling is unnecessary, because unload_toolshck and unload_winrm_ps has their own exception handling.

Reply:
No problem Please send it as a separate commit.